client/src/game/ui/drawUI.ts

```diff
@@ -1053,7 +1077,7 @@ function drawScoreArea() {
     });
     terminateButton.tooltipName = "kill";
     terminateButton.tooltipContent =
-      "Terminate the game, ending it immediately.";
+      'Vote to terminate the game (click again to cancel the vote).<br><span style="padding-left: 30px;">Double click to terminate immediately.</span>';
```

Comment:
i thought we wanted right click to terminate?

Reply:
Indeed, but it felt to me more natural to have similar events for mouse and touch: click / tap, dbl click and dbl tap. I will branch this and make a new PR, accept either.